pkg/planner/core/casetest/partition/testdata/partition_pruner_in.json

```diff
@@ -218,11 +218,11 @@
       },
       {
         "SQL": "select * from t1 where t1.a in (select b from t2 where a in (1,2)) order by a",
-        "Pruner": "t1: p0,p1; t2: p0"
+        "Pruner": "t1: all; t2: p0"
       },
       {
         "SQL": "select * from t1 where t1.a in (select b from t1 where a in (1,2)) order by a",
-        "Pruner": "t1: p0; t1: p0,p1"
+        "Pruner": "t1: all; t1: p0"
```

Comment:
I think should be `t1: p0; t1: p0`. Am I right?

Reply:
This indeed looks weird, I will investigate and see what changed...

Reply:
The pruning or plan has not changed, but when extracting the Pruner string, the strings are sorted first by table, then by partitions, so 'all' is less than 'p...', so since it is the same table, "t1: all" < "t1: p..." resulting in the need to change the order :)